api-lib/circleAdmin.ts

```diff
@@ -39,6 +48,21 @@ const middleware =
         await handler(req, res);
         return;
       }
+
+      if (sessionVariables.hasuraRole === 'api-user') {
```

Comment:
I'd think it'd be nice to have this be a constant rather than a string. `API_USER_ROLE` maybe?

Reply:
its strongly typed, so if the string was anything invalid it would throw a type error, so don't think it would be necessary in this case